src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -22,7 +22,8 @@ cell_diffusion_timescale(model) = cell_diffusion_timescale(model.closure, model.
 cell_diffusion_timescale(::Nothing, diffusivities, grid) = Inf
 
 maximum_numeric_diffusivity(κ::Number) = κ
-maximum_numeric_diffusivity(κ::NamedTuple) = maximum(κ)
+maximum_numeric_diffusivity(κ::FunctionField) = maximum(κ)
+maximum_numeric_diffusivity(κ::NamedTuple) = maximum(maximum_numeric_diffusivity.(values(κ)))
```

Comment:
```suggestion
maximum_numeric_diffusivity(κ_tuple::NamedTuple) = maximum(maximum_numeric_diffusivity(κ) for κ in κ_tuple)
```

Often good to avoid broadcasting when not necessary

Reply:
Let me know when this change is in and I will approve